sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/patch/PatchUtil.java

```diff
@@ -57,6 +58,16 @@ private static JsonSerializable cosmosPatchToJsonSerializable(CosmosPatchOperati
 
         jsonSerializable.set("operations", operations);
 
+        if(cosmosPatchSpec.getRequestOptions() != null)
+        {
+            String filterPredicate = cosmosPatchSpec.getRequestOptions().getFilterPredicate();
+            if(filterPredicate != null) {
+                if (!filterPredicate.isEmpty()) {
+                    jsonSerializable.set("condition", filterPredicate);
```

Comment:
string constant `condition` and other new constants should be defined as constants.